numba/cpython/tupleobj.py

```diff
@@ -187,6 +187,19 @@ def iternext_unituple(context, builder, sig, args, result):
         builder.store(nidx, iterval.index)
 
 
+@overload(operator.getitem)
+def getitem_literal_idx(tup, idx):
+    if not (isinstance(tup, types.BaseTuple)
```

Comment:
Can we add a note here explaining why we need this `overload`? something like the description of the PR.